docker-versions-create.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#set -x
+# Use this variable to indicate a list of branches that docker hub is watching
+branches_list=('openj9' 'another_name')
+
+function TrapExit {
+  echo "Checking out back in master"
+  git checkout master
+}
+
+trap TrapExit EXIT SIGTERM
+
+test -d ./.git || { echo ".git folder was not found. Please start this script from root directory of the project!";
+  exit 1; }
+
+# Making sure we are in master
+git checkout master
+#Updating git branch list. Cause the branch may be created from diff place
+# Are tags in use? if not - remove --tags
+git fetch --all --tags || { \
+  echo "Can't fetch changes from the remote. Permissions?"; \
+  exit 1; }
+# Updating current master from remote. Cause repo could be changed
+git merge origin/master || { echo "Can't update local master from remote repo!"; \
+  exit 1; }
```

Comment:
I think these two could be combined into a `git pull` since it's a good assumption that the local `master` branch is already associated with an upstream.